lms/resources/_js_config/__init__.py

```diff
@@ -538,3 +539,32 @@ def _sync_api(self):
             return self._blackboard_sync_api()
 
         return None
+
+    def _add_deeplinking_api(self):
+        """
+        Add the details of the "Deeplinking API" in LMS where we support deeplinking.
+
+        This API will be used by the frontend to retrieve the form fields required
+        for the deeplinking submission while on CONTENT_ITEM_SELECTION mode to store the
+        selected content on the LMS.
+        """
+        if (
+            not self._context.is_canvas
+            or self._application_instance.lti_version != "1.3.0"
+        ):
+            return
+
+        config = {
+            "data": {
+                "content_item_return_url": self._context.lti_params[
+                    "content_item_return_url"
+                ],
+            },
+        }
+
+        config["path"] = self._request.route_path("lti.deeplinking.formfields")
+        config["data"]["deep_linking_settings"] = self._context.lti_params[
+            "deep_linking_settings"
+        ]
+
+        self._config["filePicker"]["deeplinkingAPI"] = config
```

Comment:
It's strange because in the follow up PR it changes some bits only for LTI1.1. 

I'll commit your version as it makes more sense standalone and it's not a big change later.